scripts/splitControl.js

```diff
@@ -53,7 +53,10 @@ var splitControl = function(container, config) {
     let pane1 = $(children[0]);
     let pane2 = $(children[1]);
 
-    let dragBar = $("<div>").css({"background-color": theConfig.dragBarColor, flex: `0 0 ${theConfig.dragBarSize}px`});
+    let dragBar = $("<div>").css({"background-color": theConfig.dragBarColor, flex: `0 0 ${theConfig.dragBarSize}px`, display: 'flex', 'justify-content': 'center'})
+        // this div puts a row of dashes in the centre of the drag bar
+        .append($("<div>").css({border: `${theConfig.dragBarSize / 3}px dashed DimGray`, margin: `${theConfig.dragBarSize / 6}px`, flex: '0 0 6em'}));
```

Comment:
does DimGray make sense for all our themes or should we use some css that is themeable here?

Reply:
Currently the proofing interfaces and page browser and related are not themed, but I think should be themable. Anything that specifies a color in the (non-css/less) code will have to have it pulled out at some point.

It's true that the colors in the Format Preview display can be customised by the individual user, but the container cannot be.

Mind you, it works for now, and can be changed when work starts on the theming, but it would be better if it was located in the `page_interfaces.less` file, not in the `js` file.

Reply:
All of the drag bar styling could be moved to css, even the width which would avoid the new complexity of adjusting the marking to fit the drag bar.